docs/usage/configuration/config_documentation.md

```diff
@@ -533,6 +547,20 @@ listeners:
     bind_addresses: ['::1', '127.0.0.1']
     type: manhole
 ```
+Example configuration #3:
+```yaml
+listeners:
+  # Unix socket listener: Ideal for Synapse deployments behind a reverse proxy, offering lightweight interprocess communication without TCP/IP overhead, avoid port conflicts, and providing enhanced security through system file permissions.
```

Comment:
(wrap line)

Reply:
Fixed in [2f20dda](https://github.com/matrix-org/synapse/pull/15708/commits/2f20ddae4991ce5521822baefa5bb8630c9d220f)